doc/rados/configuration/auth-config-ref.rst

```diff
@@ -1,142 +1,141 @@
 .. _rados-cephx-config-ref:
 
 ========================
- Cephx Config Reference
+ CephX Config Reference
 ========================
 
-The ``cephx`` protocol is enabled by default. Cryptographic authentication has
-some computational costs, though they should generally be quite low.  If the
-network environment connecting your client and server hosts is very safe and
-you cannot afford authentication, you can turn it off. **This is not generally
-recommended**.
+The CephX protocol is enabled by default. The cryptographic authentication that
+CephX provides has some computational costs, though they should generally be
+quite low. If the network environment connecting your client and server hosts
+is very safe and you cannot afford authentication, you can disable it.
+**Disabling authentication is not generally recommended**.
 
-.. note:: If you disable authentication, you are at risk of a man-in-the-middle
-   attack altering your client/server messages, which could lead to disastrous
-   security effects.
+.. note:: If you disable authentication, you will be at risk of a
+   man-in-the-middle attack that alters your client/server messages, which
+   could have disastrous security effects.
 
-For creating users, see `User Management`_. For details on the architecture
-of Cephx, see `Architecture - High Availability Authentication`_.
+For information about creating users, see `User Management`_. For details on
+the architecture of CephX, see `Architecture - High Availability
+Authentication`_.
 
 
 Deployment Scenarios
 ====================
 
-There are two main scenarios for deploying a Ceph cluster, which impact
-how you initially configure Cephx. Most first time Ceph users use
-``cephadm`` to create a cluster (easiest). For clusters using
-other deployment tools (e.g., Chef, Juju, Puppet, etc.), you will need
-to use the manual procedures or configure your deployment tool to
-bootstrap your monitor(s).
+How you initially configure CephX depends on your scenario. There are two main
+scenarios for deploying a Ceph cluster.  If you are a first-time Ceph user, you
+should probably take the easiest approach: using ``cephadm`` to deploy a
+cluster. But if your cluster uses other deployment tools (for example, Chef,
+Juju, or Puppet), you will need either to use the manual deployment procedures
```

Comment:
Add Ansible to the list, I have to believe that more people use it to deploy Ceph than use Chef.